docs/source/federated-types/federated-directives.mdx

```diff
@@ -32,13 +32,30 @@ extend schema
 
 This example subgraph schema uses `@primaryKey` for the federated directive usually named [`@key`](#key).
 
-## Managing entities
+### `@link`
```

Comment:
its somewhat odd that `@link` definition appears after mentioning it above for importing and renaming the directives - maybe it should be moved up?

Reply:
also should we add sections about namespacing? i.e. if you don't explicitly import types then they should be namespaced with `@federation__` prefix (unless custom one is provided)

Reply:
I'd prefer to leave the `@link` section below importing and renaming, because those sections are critical setup for making use of any other directive on the page, whereas the `@link` section is more reference like the rest of the directives.

Yes agreed on the namespacing, I'll add that